testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/AbstractKeycloakTest.java

```diff
@@ -116,6 +117,9 @@ public abstract class AbstractKeycloakTest {
 
     protected Logger log = Logger.getLogger(this.getClass());
 
+    @Rule
+    public Timeout globalTimeout = Timeout.seconds(90);
```

Comment:
Setting the global time to 90 seconds might impact testing in Jenkins environment. I would suggest to run a test pipeline with these changes and see how it performs.